City-of-Pori/pori-kada
# PORI-199 person card

To test:

1. drush fra -y && drush cc all
2. drush fra -y && drush cc all (you need to revert features twice for some reason)
3. Make sure you have a person type content. If not, go add two with a lot of information.
4. Go to edit any page type content. Under the "Additional information" tab there should be "Contact person" field as the last item. Add here the two you just created or any of the persons that you have on your database.
5. Go to see the page and make sure the contact persons are nicely lined up under the additional information block and with layout that follows this: https://drive.google.com/drive/u/0/folders/0B7ZHsug7PcEDVDBNTG51a0RIMzQ
